internal/image/digest_selector.go

```diff
@@ -0,0 +1,82 @@
+package image
```

Comment:
This file used to be `digest_tag_selector.go`. With the rename of many variables and functions, GitHub seems not to recognize this as a renamed file and thinks it is new.

This applies to all the selectors.